### ISSUE TITLE
Code not available online

### ISSUE THREAD

**Issue by hassnain360** [bwang40/OV7670_NEXYS4DDR_HDL#1]
Hi, the code that your article refers to is missing from the main source. Could you please post your code for interfacing OV7670 with Nexys 4 board ?

**Comment by AmarVLSI** [comment]
Yeah Codes are missing and constraints files are also missing. Please add the constraints file.

**Comment by AmarVLSI** [comment]
> Hi, the code that your article refers to is missing from the main source. Could you please post your code for interfacing OV7670 with Nexys 4 board ?

were you got the code?
